include/picongpu/fields/incidentField/profiles/ExpRampWithPrepulse.hpp

```diff
@@ -191,7 +191,7 @@ namespace picongpu::fields::incidentField
                 {
                     auto const runTimeShifted = time + Unitless::time_start_init;
                     auto const phase = Unitless::w * runTimeShifted + Unitless::LASER_PHASE + phaseShift;
-                    return math::sin(phase) * Unitless::AMPLITUDE * Envelope::getEnvelope(runTimeShifted);
+                    return math::cos(phase) * Unitless::AMPLITUDE * Envelope::getEnvelope(runTimeShifted);

```

Comment:
why this change to `cos`?

Reply:
We can change it back, I guess, since the functor is going to be depreciated with this PR. But cos is the usual definition I guess? It is also cos in `GaussianPulse` and the two implementations were shifted in phase because fo that.

Reply:
since it is deprecated soon - no need to change